src/navigation/Router.tsx

```diff
@@ -6,14 +6,15 @@ import {
   Route,
   createRoutesFromElements,
 } from 'react-router-dom';
-import { PageOneContainer } from '../pages/PageOne/PageOneContainer';
+import { ProductsPageContainer } from '../pages/ProductsPage/ProductsPageContainer';
 
 export const router = createBrowserRouter(
   createRoutesFromElements(
     <Route path="/" element={<MainLayout />}>
       <Route path="/" index element={<WelcomePage />} />
-      <Route path="/page1" element={<PageOneContainer/>} />
+      <Route path="/productsPage" element={<ProductsPageContainer/>} />
```

Comment:
I don't think we need to add 'Page', just  products will be sufficient

Reply:
Route path has been renamed to "/products"